### PR TITLE
SONARJAVA-1005 Fix FP on loop incrementers

### PR DESCRIPTION
**Pull request by m-g-sonar**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

**Comment by SonarTech** [comment]
SonarQube analysis reported no new issues.

File: java-checks/src/test/files/checks/ForLoopIncrementAndUpdateCheck.java [hunk 1]
```diff
@@ -1,33 +1,48 @@
...
+    for (i = 0; k< 10 ; i++, j++, l++) { // Noncompliant {{This loop's stop condition tests "k" but the incrementer updates "i, j, l".}}
     }
-    for (i = 0; k< 10 ; i++, j++, l++) { //Noncompliant
+    for (i = 0; k< 10 && l<10; i++, j++) { // Noncompliant {{This loop's stop condition tests "k, l" but the incrementer updates "i, j".}}
     }
-    for (i = 0; k< 10 && l<10; i++, j++) { //Noncompliant
+    for (i = 0; k< 10; i+= 2) { // Noncompliant {{This loop's stop condition tests "k" but the incrementer updates "i".}}
     }
-    for (i = 0; k< 10; i+= 2) { //Noncompliant
+    for (i = 0; i< 10; i+= 2) { // Compliant
```
Comment (benzonico) [inline java-checks/src/test/files/checks/ForLoopIncrementAndUpdateCheck.java]: Ins't that a removal of a test ?

**Comment by benzonico** [comment]
Except comment, LGTM

File: java-checks/src/test/files/checks/ForLoopIncrementAndUpdateCheck.java (hunk 1 above)
Comment (m-g-sonar) [inline java-checks/src/test/files/checks/ForLoopIncrementAndUpdateCheck.java]: No, look at the (new) line 19

**Pull request closed without merging**